src/rust/engine/rule_graph/README.md

```diff
@@ -97,22 +98,79 @@ But as the set of rules in Pants has grown, more cycles have made their way into
 
 The special case mentioned above to break out of monomorphize if it is not converging allows us to successfully construct graphs for most of these dozen-node cases (relatively quickly: 4000 node visits to construct a 450 node output graph), but larger cases can take exponentially longer, and the special case causes us to break out with errors in cases that should be resolvable.
 
-### Issues
+### Example
 
-#### Monotonicity vs Global Information
+One minimal problematic testcase (`fn wide_cycle` in `src/rust/engine/rule_graph/src/tests.rs`) looks like:
```

Comment:
Maybe add an example of a non-problematic test case. For example, show what the graph looks like for the Fibonannaci example.

Also, consider adding pictures of the visualized graph. Readme.io allows you to upload pictures.